### ISSUE TITLE
Unable to view options pane on Classic

### ISSUE THREAD

**Issue by jtferm** [Backupiseasy/ThreatPlates#309]
Classic Era and the Season of Mastery Beta

I was not able to load the options pane via the `/tptp` command nor in the interface options window. On load, it throws a couple of lua errors:

```
Message: Interface\AddOns\TidyPlates_ThreatPlates\Core.lua:376: attempt to call method 'AddToBlizOptions' (a nil value)
Time: Sat Oct 30 14:05:45 2021
Count: 1
Stack: Interface\AddOns\TidyPlates_ThreatPlates\Core.lua:376: attempt to call method 'AddToBlizOptions' (a nil value)
[string "@Interface\AddOns\TidyPlates_ThreatPlates\Core.lua"]:376: in function <Interface\AddOns\TidyPlates_ThreatPlates\Core.lua:333>
[string "=[C]"]: ?
[string "@Interface\AddOns\TidyPlates_ThreatPlates\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:70: in function <...ates_ThreatPlates\Libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Interface\AddOns\TidyPlates_ThreatPlates\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:498: in function `InitializeAddon'
[string "@Interface\AddOns\TidyPlates_ThreatPlates\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:623: in function <...ates_ThreatPlates\Libs\AceAddon-3.0\AceAddon-3.0.lua:615>

Locals: <none>
```
...which appears to be why the `/tptp` command doesn't get registered, and:

```
Message: ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:568: attempt to call method 'SetBackdrop' (a nil value)
Time: Sat Oct 30 13:57:18 2021
Count: 1
Stack: ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:568: attempt to call method 'SetBackdrop' (a nil value)
[string "@Interface\AddOns\TidyPlates_ThreatPlates\Libs\AceConfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua"]:568: in main chunk

...
```
...which happens within the AceConfig library, and appears to be why the `AddToBlizOptions` method doesn't get defined and can't be used, causing the first error.
(I can include the rest of the dump if it's helpful, but I don't think it adds much here)

When the client updated to version 9.0.1 it changed how the backdrop api worked for frames, [outlined here](https://github.com/Stanzilla/WoWUIBugs/wiki/9.0.1-Consolidated-UI-Changes#lua-changes). It's a pretty simple change just to get it working, but I'm a complete Lua noob so I'm not sure the best way to make a change since it's in a library dependency.

**Comment by Backupiseasy** [comment]
Thanks for highlighting and fixing this (in the pull request). I updated the internal Ace3 library to a newer version which fixes the issue. See here: https://www.wowace.com/projects/ace3/issues/572. That should be the same error. I'll upload a new release today hopefully.

**Comment by Backupiseasy** [comment]
You can find a version to test (for Classic) here: https://www.curseforge.com/wow/addons/tidy-plates-threat-plates/files/3511434

**Comment by Backupiseasy** [comment]
Should be fixed with release 10.2.13.
